### DISCUSSION TITLE
Antivirus: ESET and other identify virus in setup file

### DISCUSSION THREAD

**Discussion by rawbyne** [amidaware/tacticalrmm#1504] [General]
Hi there, 

as you can see under the following link, some AV-Scanner - as mine (ESET) - do not trust. How to deal with it?

Version 2.4.3
https://www.virustotal.com/gui/file/65078299107ccfc5764dceb032a8fd3c4fa3bb065ab7d5b8d7e0ad3c58e4c239/detection

Version 2.4.6
https://www.virustotal.com/gui/file/746e91c29d01ddc6e465548234c97bcfe774e344fe53ad38d99ff912f23de1db/detection

**Comment by silversword411** [comment]
https://docs.tacticalrmm.com/av/

**Comment by rawbyne** [comment]
Hi, I know that document. But it is not related to that problem. These exclusions have been made. But if a customer is downloading that tool by saveing it under a custom path, it will be shown up as an virus file. Would you install that file from your IT-Professional, if your AV shows up, that is is definitly a virus?

That we have to make file exclusions or folder exclusions is right. But the downloadable custom agent has to be shown as virus-free.

**Reply by wh1te909** [reply]
Trmm is obviously not a virus. If your AV thinks it's a virus, then that's problem with your AV and you need to complain to them for making false accusations :) There isn't anything we can do on our end except report false positive which we encourage everyone to do.

**Reply by rawbyne** [reply]
I know. I will wait on reply from eset. But other AV scanners should also be informed about that. By now 7 avs (also Kaspersky) meens it is a virus file. Not good...

**Reply by dinger1986** [reply]
Doesn't mean it's a virus, it's a false positive!

**Reply by rawbyne** [reply]
I know...

**Reply by rawbyne** [reply]
Wow. It is getting worser and I do not have the menpower to evaluate all AV-Scanners. Last week there were 7 AV-Scanners, which identified the setup-file as a virus. Now virus-total sais: 22 AV-Scanners. On my Side, I informed ESET about false positive...

Take a look: https://www.virustotal.com/gui/file/65078299107ccfc5764dceb032a8fd3c4fa3bb065ab7d5b8d7e0ad3c58e4c239/detection

**Reply by dinger1986** [reply]
Are you checking code signed or non code signed agents?

**Reply by rawbyne** [reply]
I use code signing in m tactical rmm. If new agents via "manage deployments" are code signed then, too....

**Comment by dinger1986** [comment]
code signing helps

Also we normally onboard using another piece of software and then install tactical.

Obviously the nature of what it is (Remote Control Software) leaves the opportunity for bad actors to use it so AVs will flag it up sometimes as they will any remote control software.

Best way is to report the false positive.

**Reply by rawbyne** [reply]
At ESET I reported it as false positive this day. I am hoping to get reply soon.

**Reply by rawbyne** [reply]
Hi again. Nearly all of my customers are shown as offline because of the virus titulation. What do I have to do to get rid of these problems. @bbrendon shown up a link of virustotal, where he do not have that problems. So what is the difference between his and my file? I do not understand, what to do. Please help.

**Reply by silversword411** [reply]
Hey there. So has ESET fixed their problem, and yours and responded to the false positive report? Or are they just ignoring you?

**Reply by rawbyne** [reply]
until now I do not got any response

**Reply by silversword411** [reply]
Discord is a better live communication. 

ESET ignoring their customers, you should get someone on the phone and raise hell. A week without response for an AV company is neglect, or incompetence. Not what you'd expect for support for paying customers.

**Comment by bbrendon** [comment]
Dont use deployment links?
https://www.virustotal.com/gui/file/9135e32aee27343e733c56d2d7b82fe2f8442588c999091dd654da51cf13da71

**Reply by dinger1986** [reply]
I checked the agent earlier, didn't want to try a direct install just incase it exposed my token 😂

**Reply by rawbyne** [reply]
sorry, but I do not understand the difference between my file and your file. Your file is not shown up as a virus, but mine. How do I have to configure my system?
